azure-pipelines.yml

```
@@ -87,6 +87,7 @@ jobs:
   - pwsh: |
       .\validateWorkerVersions.ps1
     displayName: 'Validate worker versions'
+    condition: eq(variables['validateWorkerVersions'], 'true')
```

I'm fine with adding a conditional, but the task should be enabled by default. Maybe something like this instead:

```suggestion
    condition: ne(variables['skipWorkerVersionValidation'], 'true')
```